alembic/versions/0a1c5d9b7e44_lecture_video_runtime_control_and_.py

```
@@ -0,0 +1,233 @@
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "0a1c5d9b7e44"
+down_revision: Union[str, None] = "4db0bf59f8c2"
```

Ruff (0.15.5)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)